stock-filters/ChirpVillage/Biomes/BiomeSettings.py

```diff
@@ -121,19 +139,23 @@
         "beam": blocks['Piller Quartz Block'],
         'hedge': blocks['Snow Block'],
         'roof': blocks['Quartz Block'],
-        'crop': blocks['Potatoes']
+        'crop': blocks['Potatoes'],
+        'soil': blocks['Dirt'],
+        'bridge': blocks['Piller Quartz Block']
     },
     "Mountains": {
-        "wall": blocks['Oak Wood Planks'],
+        "wall": blocks['Bricks'],
         "fence": blocks['Oak Fence'],
         "floor": blocks['Stone Bricks'],
-        "road": blocks['Moss Stone'],
+        "road": blocks['Stone Bricks'],
         "door": blocks['Oak Door Block'],
         'window': blocks['Glass Pane'],
         "beam": blocks['Oak Wood'],
         'hedge': blocks['Oak Leaves'],
-        'roof': blocks['Mossy Stone Bricks'],
-        'crop': blocks['Wheat']
+        'roof': blocks['Stone Bricks'],
+        'crop': blocks['Wheat'],
+        'soil': blocks['Dirt'],
```

Comment:
This is for the farm issue in the desert right?

Reply:
yup!